dipy/reconst/eudx_direction_getter.pyx

```diff
@@ -26,6 +26,15 @@ cdef class EuDXDirectionGetter(DirectionGetter):
         self.qa_thr = 0.0239
         self.ang_thr = 60
         self.total_weight = .5
+        self.sphere = None
```

Comment:
This seems to be the source of the test error. Could you move these out to the `PeaksAndMetrics` python class instead?